electron/src/routes.ts

```diff
@@ -324,25 +324,32 @@ export function getPlatformApp(data: { id: string }, callback) {
             callback(new Error("Cannot fetch an app, you are not connected to any platform."));
         }
 
-        swapController.exists(credentials.id + "/" + data.id, (err, exists) => {
+        if (!data.forceFetch) {
+            swapController.exists(credentials.id + "/" + data.id, (err, exists) => {
 
-            if (err) {
-                callback(err);
-                return;
-            }
+                if (err) {
+                    callback(err);
+                    return;
+                }
 
-            if (exists) {
-                swapController.read(credentials.id + "/" + data.id, callback);
-                return;
-            }
+                if (exists) {
+                    swapController.read(credentials.id + "/" + data.id, callback);
+                    return;
+                }
 
+                const api = new SBGClient(credentials.url, credentials.token);
+                api.apps.get(data.id).then(response => {
```

Comment:
Although api.apps.get and getApp do the same don't use both in the same function :)